src/button/component.jsx

```diff
@@ -993,9 +993,22 @@ export const Button : Component<ButtonOptions> = create({
                     shape:        BUTTON_SHAPE.PILL,
                     size:         BUTTON_SIZE.SMALL,
                     label:        BUTTON_LABEL.CHECKOUT,
-                    fundingicons: false
+                    fundingicons: false,
+                    layout:       BUTTON_LAYOUT.HORIZONTAL
                 };
             },
+            
+            decorate(style : Object) : Object {
+                const { label, layout } = style;
```

Comment:
Should we do `layout = BUTTON_LAYOUT.HORIZONTAL`?

Reply:
ok